### DISCUSSION TITLE
[用户脚本]在github讨论中使用米游社表情包

### DISCUSSION THREAD

**Discussion by Dave-12138** [share121/inter-knot#4639] [教程]
我使用脚手架工具重新编写了在github讨论中使用米游社表情包的用户脚本
![示例图片](https://github.com/Dave-12138/miyoushe-emoticon-in-discussion/raw/main/readme/image2.png)
要使用该脚本，你至少需要一个**脚本加载器**(比如[TamperMonkey](https://www.tampermonkey.net/))或**自带脚本加载器的浏览器**(比如[via](https://viayoo.com/zh-cn/))。
然后从[这个链接](https://dave-12138.cn/static/Tampermonkey/miyoushe-emot-to-github.user.js)或者[这个链接](https://dave-12138.github.io/miyoushe-emoticon-in-discussion/miyoushe-emot-to-github.user.js)安装脚本。如果脚本管理器问你是否允许跨域，点击**总是允许该域名**或长得差不多的允许选项。
现在脚本使用 vue 渲染界面，添加了当前选择的表情包分类提示、鼠标悬浮在表情包上时的表情包名称提示。
现在你可以在楼中楼中使用该脚本了——你点击哪个输入框，表情包选择界面就跟到哪。
[仓库地址](https://github.com/Dave-12138/miyoushe-emoticon-in-discussion)

> - 所以第一页的表情包为什么有五个阿姬倒地？
> - 不知道，反正老米的API给了我五个一模一样但链接不同名称也不一样的阿姬倒地

**Comment by ID104772508** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >感谢大佬！！

**Reply by MQZCZBCK** [reply]
<img src="https://img-static.mihoyo.com/communityweb/upload/613a2b262af0319edde21587b88a9c6e.png" alt="米游姬-吃瓜" width="100" >

**Comment by anewperson001** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/ef40ae21777834eaf56bac6b15dc2f7d.png" alt="米游姬 抱抱" width="75" >感谢大佬！！

**Reply by starshine4233** [reply]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >u

**Comment by 0xSelenicDove** [comment]
Safari用Stay，拦截100不工作

**Reply by 0xSelenicDove** [reply]
而且Stay免费，拦截100这个功能收费<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/d364cbf606b633be67a1536289347556_8458169816302857767.png" alt="时光交响之章表情包10" width="75" >

**Reply by 0xSelenicDove** [reply]
iPhone、iPad、mac用户看这里！求置顶！<img src="https://upload-bbs.miyoushe.com/upload/2023/02/03/a16450f92757f9d45588a8541fbbab09_4901912883842187271.gif" alt="米游姬-感谢" width="75" >

**Comment by mztxy229** [comment]
感谢感谢已经用上了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by zxdgle** [comment]
可以<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Comment by hdjsjjdgrhj** [comment]
谢谢大佬，已经用上了<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by xingyun-kb** [comment]
感谢<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/4b4fb49d4bd402c530600e758ebb06bd_5477506439621383475.png" alt="阿姬开心" width="75" >

**Comment by nhb-bcd** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8a1c5e858456b720f2c3c45982729628_6809351812774032443.gif" alt="米游姬 干杯" width="75" >
干杯

**Comment by fjhsjdhfhdjjw** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/10/10/86dcf94d464b0b895123976278481ace_3780637826241568223.png" alt="阿库_擦汗" width="75" >试一下

**Comment by fjhsjdhfhdjjw** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >试一下

**Comment by fjhsjdhfhdjjw** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/7cb87b286c3d062300d31224508c0c9e_8768116406867668880.png" alt="丹恒 惊" width="75" >怎么发不到绳网

**Comment by dudu-ke** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by linzhiyao123** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8701a3b5fc90a6c37ba5491913b8218f_2209157026344411896.png" alt="布洛妮娅 开心鸭" width="75" >试一下

**Comment by Psinary** [comment]
![](https://img-static.mihoyo.com/communityweb/upload/b359242ee7d6676fc02afc68552eb311.png)

**Comment by faatong** [comment]
光发表情发送按钮是灰色的<img src="https://bbs-static.miyoushe.com/static/2023/11/16/2544d70667e060623d4b7702cd6bb3b9_8763505029750571668.png" alt="可琳-紧张" width="75" >

**Reply by Dave-12138** [reply]
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >不 可 发 送 单 个 表 情

**Comment by IceThunder** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" > 不得不说，简单粗暴

**Comment by HoshinoHanashi** [comment]
学会啦<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by cuijiaqi0406** [comment]
我还是不会用啊，怎么办（手动狗头）<img src="https://bbs-static.miyoushe.com/static/2023/11/16/874ff589832ff66517ac5d2999b75e40_5711211283435347869.png" alt="厨师邦布-啊？" width="75" >

**Comment by Lun603** [comment]
大佬你太棒囉~<img src="https://img-static.mihoyo.com/communityweb/upload/bf7e7cae6c42a727faa3d5e76b4a4c1c.png" alt="帕姆_比心" width="75" >

**Comment by Anai136** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >谢谢大佬

**Comment by Anai136** [comment]
试一下<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by buroro** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/3b2e51180d866548d11c650a2dcf6d43.png" alt="三月七_暗中观察" width="75" >蟹蟹佬 #

**Comment by Zou-pig** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/43338db72e2bb898d13a84c2fc302cb8_7770278814004091890.png" alt="阿姬调查" width="75" >好耶谢谢大佬

**Comment by zwz2732629957** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/02/02/6f42af07b1038edef910c8450bb6cd40_7801347074520422045.png" alt="赫丽娅-可靠" width="75" >

**Comment by coolwide** [comment]
现在手机端可以咩？

**Reply by Dave-12138** [reply]
手机浏览器吗 能用，但对edge canary之类的可能需要自己打开开发者模式

**Reply by coolwide** [reply]
现在不是有安卓的app咩，我不知道怎么在app上搞捏，大佬有办法吗

**Reply by Dave-12138** [reply]
GitHub APP就没办法了，一个用户脚本又不能注入到APP里

**Comment by ChaoKaiADG** [comment]
Via浏览器提示：下载依赖失败，请查明原因，蟹蟹<img src="https://bbs-static.miyoushe.com/static/2024/10/24/0296f44f6aa17344294e4ccca4829b5a_2147609872749895683.png" alt="悠闲01" width="75" >

**Reply by Dave-12138** [reply]
换了个知乎的cdn，这下应该不会获取不到了吧<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Comment by Hoyoer** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >成功了！

**Comment by MO-hy** [comment]
！

**Comment by woaixier** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/12/04/aa5a872696c638173ddd3fe4a47f8239_2657736912141711400.png" alt="星期日 讲" width="75" >谢谢大佬也是用上了

**Comment by zgyhim** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/12/04/2b4d005c528ee4b51da394899f8025fd_3529972885399131670.png" alt="银狼 干嘛" width="75" > good

**Comment by 1920668427** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/10/24/b364c5784e16b15f53af9db191666b4a_936313514384740306.png" alt="TADA" width="75" >牛啊大佬

**Comment by tanchideshaohuaya** [comment]
用上了<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/4b4fb49d4bd402c530600e758ebb06bd_5477506439621383475.png" alt="阿姬开心" width="75" >

**Comment by Daytime-meteors** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/10/21/a23575defbaa22a609b71b515b343953_3443100384601092130.png" alt="德丽莎-我没睡" width="75" >知道了，知道了。<img src="https://bbs-static.miyoushe.com/static/2024/10/21/009f9c92fc0f452a35f5cee54d730fc6_3020497492997022606.png" alt="德丽莎-下班" width="75" >

**Comment by KevinyeeOfficial** [comment]
<img src="https://bbs-static.miyoushe.com/static/2025/02/07/5b5bed2da952a02dd64f5d454787e540_2296647617608585144.png" alt="星见雅-满足" width="75" >用上了！<img src="https://bbs-static.miyoushe.com/static/2025/02/07/5b5bed2da952a02dd64f5d454787e540_2296647617608585144.png" alt="星见雅-满足" width="75" >

**Comment by dao42-cq** [comment]
在手机上要怎么弄啊😦

**Reply by XunSu1028** [reply]
手机可以用edge的“篡改猴”扩展，剩下的就按他说的网址正常安装脚本就可以了。
<img src="https://bbs-static.miyoushe.com/static/2024/09/29/28c8a4bcf98f2da1a20937ed62531487_6725617669327375219.png" alt="本-自信" width="75" >
